### PR TITLE
Add check of DRM master.

### PR DESCRIPTION
**Pull request by Mai-Matsuura**
# Description
I added a check if the process has become a DRM master after opening the DRM device.
If it fails to become a DRM master, the application will terminate.

# Related issues
#15

### CODE REVIEW

**Comment by Mai-Matsuura** [comment]
@HidenoriMatsubayashi
Can I request you to review?

File: src/flutter/shell/platform/linux_embedded/surface/native_window_drm.cc [hunk 1]
```diff
@@ -27,6 +27,12 @@ NativeWindowDrm::NativeWindowDrm(const char* deviceFilename) {
     LINUXES_LOG(ERROR) << "Couldn't open " << deviceFilename;
     return;
   }
+
+  if (!drmIsMaster(drm_device_)) {
```
Comment (HidenoriMatsubayashi) [inline src/flutter/shell/platform/linux_embedded/surface/native_window_drm.cc:31]: Did you test on Raspberry Pi4? Are there any version restrictions?

File: src/flutter/shell/platform/linux_embedded/surface/native_window_drm.cc [hunk 2]
```diff
@@ -27,6 +27,12 @@ NativeWindowDrm::NativeWindowDrm(const char* deviceFilename) {
...
+    LINUXES_LOG(ERROR) << "Failed to become a DRM master";
```
Comment (HidenoriMatsubayashi) [inline src/flutter/shell/platform/linux_embedded/surface/native_window_drm.cc:32]: The embedder couldn't become the DRM master. Please confirm if another display backend such as X11 and Wayland is not running.

**Review by HidenoriMatsubayashi — COMMENTED** [review]
Check my comments.

File: src/flutter/shell/platform/linux_embedded/surface/native_window_drm.cc (hunk 1 above)
Comment (Mai-Matsuura) [reply src/flutter/shell/platform/linux_embedded/surface/native_window_drm.cc:31]: I tested on Raspberry Pi4 and it worked fine on the DRM backend.
[According to the release information](https://lists.x.org/archives/xorg-announce/2019-April/002992.html), `drmIsMaster()` was added in libdrm 2.4.98.

**Comment by Mai-Matsuura** [comment]
@HidenoriMatsubayashi 
I fixed the code. Please re-review.

**Comment by HidenoriMatsubayashi** [comment]
LGTM

**Pull request merged**